src/utils/qr.ts

```diff
@@ -1,4 +1,4 @@
-import * as qrcode from 'qrcode-generator'
+import qrcode from 'qrcode-generator'
```

Comment:
There was a conflict between builds concerning this line. I added `"esModuleInterop": true` to the TypeScript config which fixed the issue.